R/RcppExports.R

```diff
@@ -5,7 +5,6 @@
 #' @param time vector of time values
 #' @param dv concentration measurements
 #' @param range vector of min and max value of the partial auc range
```

Comment:
I reverted these changes because 1) it looks unrelated and 2) the CI would have failed with `set_bins_cpp` exported but no Rd checked in.

f624a50 (RcppExports: revert recent changes, 2025-06-09)